WebDevStudios/wd_s
# Updates package versions

### DESCRIPTION ###
Updates npm package versions.

### STEPS TO VERIFY ###
1. Checkout
2. Run `npm outdated` to see if there are any outdated packages (should be none at this point)

### DOCUMENTATION ###
None required for this update.
